plugins/inputs/github/github.go

```diff
@@ -144,6 +147,17 @@ func (g *GitHub) Gather(acc telegraf.Accumulator) error {
 			tags := getTags(repositoryInfo)
 			fields := getFields(repositoryInfo)
 
+			if g.AdditionalFields {
+				// Pull request properties
+				prFields, err := g.getPullRequestFields(ctx, owner, repository)
+				if err != nil {
+					acc.AddError(err)
+				}
+				for k, v := range prFields {
+					fields[k] = v
```

Comment:
It may happen that `prFields` is not empty when `err != nil`. Do you still want to get these fields in case of error?